scripts/build_renewable_profiles.py

```diff
@@ -603,10 +603,18 @@ def create_scaling_factor(
             columns={"x": "lon", "y": "lat", "country": "countries"}
         ).loc[bus_in_hydrobasins, ["lon", "lat", "countries", "shape_id"]]
 
-        resource["plants"]["installed_hydro"] = [
-            True if (bus_id in hydro_ppls.bus.values) else False
-            for bus_id in resource["plants"].index
-        ]
+        if snakemake.params.alternative_clustering == False:
+            resource["plants"]["installed_hydro"] = [
+                True if (bus_id in hydro_ppls.bus.values) else False
+                for bus_id in resource["plants"].index
+            ]
+
+        ### TODO: quickfix. above case and the below case should by unified
+        if snakemake.params.alternative_clustering == True:
+            resource["plants"]["installed_hydro"] = [
+                True if (bus_id in hydro_ppls.region_id.values) else False
+                for bus_id in resource["plants"].shape_id.values
+            ]
```

Comment:
```suggestion
        # TODO: these cases shall be fixed by restructuring the alternative clustering procedure
        if snakemake.params.alternative_clustering == False:
            resource["plants"]["installed_hydro"] = [
                True if (bus_id in hydro_ppls.bus.values) else False
                for bus_id in resource["plants"].index
            ]
        else:
            resource["plants"]["installed_hydro"] = [
                True if (bus_id in hydro_ppls.region_id.values) else False
                for bus_id in resource["plants"].shape_id.values
            ]
```

What about this?
If @ekatef agrees we could go for it and close the PR :)

I'd avoid a duplicated if-if case for clarity but I think it is good to go for me :)

Reply:
@davide-f agree, and we have discussed with @energyLS that some slight improvements could make sense to avoid duplication. But I think the most important part is actually finding the reason for issues and having a fix for it. 

So, I'd go for the suggested changes if it doesn't require much time. Otherwise, feel free to merge as-is 🙂